### PR TITLE
Added listing user albums feature

### PR DESCRIPTION
**Pull request by tagonzalez**
## Summary

Added listing user albums feature

## Trello Card

https://trello.com/c/xaqTfonH/26-listar-todos-los-albumes-comprados

### CODE REVIEW

File: test/controllers/users.test.js [hunk 1]
```diff
@@ -188,3 +193,60 @@ describe('usersController.listAllUsers', () => {
...
+          .set('Authorization', token)
+          .expect(200)
+          .then(response => {
+            expect(response.body).toHaveProperty('user_albums');
+            const responseUserAlbums = response.body.user_albums;
+            expect(responseUserAlbums.length).toBe(userAlbums.length);
+            return responseUserAlbums.forEach(responseUserAlbum => {
+              expect(responseUserAlbum).toHaveProperty('title');
+              expect(responseUserAlbum).toHaveProperty('created_at');
+              return expect(responseUserAlbum).toHaveProperty('updated_at');
```
Comment (arinaldi118) [inline test/controllers/users.test.js]: Why does `expect` have a `return`?

File: test/controllers/users.test.js [hunk 2]
```diff
@@ -188,3 +193,60 @@ describe('usersController.listAllUsers', () => {
...
+            .set('Authorization', token)
+            .expect(200)
+            .then(response => {
+              expect(response.body).toHaveProperty('user_albums');
+              const responseUserAlbums = response.body.user_albums;
+              expect(responseUserAlbums.length).toBe(userAlbums.length);
+              return responseUserAlbums.forEach(responseUserAlbum => {
+                expect(responseUserAlbum).toHaveProperty('title');
+                expect(responseUserAlbum).toHaveProperty('created_at');
+                return expect(responseUserAlbum).toHaveProperty('updated_at');
```
Comment (arinaldi118) [inline test/controllers/users.test.js]: idem

**Review by arinaldi118 — CHANGES_REQUESTED** [review]

File: test/controllers/users.test.js (hunk 2 above)
Comment (tagonzalez) [reply test/controllers/users.test.js]: Forgot that it wasn't necessary, removed in latest commit

File: test/controllers/users.test.js (hunk 1 above)
Comment (tagonzalez) [reply test/controllers/users.test.js]: Forgot that it wasn't necessary, removed in latest commit

**Review by arinaldi118 — APPROVED** [review]

**Pull request merged**